backend/cmd/server/repository/resources/graphs/auth_flow_config_basic.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
Shall we revert removing these flow definition files? I believe this is unintentional